securesystemslib/signer/_gcp_signer.py

```diff
@@ -173,6 +173,10 @@ def _get_hash_algorithm(public_key: Key) -> str:
             # nistp256 uses sha-256, nistp384 uses sha-384
             bits = public_key.scheme.split("-nistp")[-1]
             algo = f"sha{bits}"
+        else:
+            raise exceptions.UnsupportedAlgorithmError(
+                f"Unsupported key type {public_key.keytype} in key {public_key.keyid}"
+            )
```

Comment:
Looks like, you'll also land here if keytype is rsa.

Reply:
oops